dashboard/app/helpers/users_helper.rb

```diff
@@ -110,7 +111,7 @@ def summarize_user_progress_for_all_scripts(user)
                 result: ul.try(:best_result) || 0,
                 submitted: submitted ? true : nil,
                 readonly_answers: readonly_answers ? true : nil,
-                paired: ul.paired? ? true : nil
+                paired: (paired_uls.include? ul.id) ? true : nil
```

Comment:
@ashercodeorg Do we care about this hound error? It keeps popping up on all the merges (DTT, DTP). Is Hound running different rules than the git commit hook?